python/dgl/graphbolt/minibatch.py

```diff
@@ -126,89 +126,101 @@ class MiniBatch:
     all node ids inside are compacted.
     """
 
-    def to_dgl_graphs(self):
-        """Transforming a data graph into DGL graphs necessitates constructing a
+    def to_dgl_blocks(self):
+        """Transforming a `MiniBatch` into DGL blocks necessitates constructing a
         graphical structure and assigning features to the nodes and edges within
-        the graphs.
+        the blocks.
         """
         if not self.sampled_subgraphs:
             return None
 
+        reverse_row_node_ids = self.sampled_subgraphs[0].reverse_row_node_ids
+        assert (
+            reverse_row_node_ids is not None,
+            "Missing `reverse_row_node_ids` in sampled sub graph.",
+        )
+        reverse_column_node_ids = self.sampled_subgraphs[
+            0
```

Comment:
not `-1`? always fetch from first subgraph?

Reply:
_1 is the `innermost`, 0 is `outermost`